roles/kubernetes/node/templates/kubelet.kubeadm.env.j2

```diff
@@ -28,7 +28,9 @@ KUBELET_HOSTNAME="--hostname-override={{ kube_override_hostname }}"
 {% endif %}
 --enforce-node-allocatable={{ kubelet_enforce_node_allocatable }} \
 --client-ca-file={{ kube_cert_dir }}/ca.crt \
---rotate-certificates=true \
+{% if kubelet_rotate_certificates %}
+--rotate-certificates \
```

Comment:
should this have the `=true` like you previously had?

Reply:
not necessary
follow this docs:
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#options
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation
https://kubernetes.io/docs/tasks/tls/certificate-rotation/#enabling-client-certificate-rotation